src/pages/main/components/SortBox.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Typography } from '@mui/material';
+import FilterChip from './FilterChip';
+import { SortBoxProps } from '../../../types/accommodation.types';
+
+const sortStyles = {
+  display: 'flex',
+  flexWrap: 'wrap',
+  columnGap: '1rem',
+  rowGap: '1rem',
+  marginY: '0.5rem',
+  marginX: 'auto',
+  width: {
+    '2xl': '85%',
+  },
+};
+
+const SortBox = ({ title, options, minItem, setValue, name }: SortBoxProps) => {
+  const handleSelect = (id: number | string) => {
+    setValue((prev) => ({ ...prev, [name]: id }));
+  };
+
+  return (
+    <Box>
+      <Typography>{title}</Typography>
+      <Box sx={sortStyles}>
+        {options.map((item) => (
+          <FilterChip
+            key={item.id}
+            item={item}
+            minItem={minItem}
+            handleSelect={() => handleSelect(item.id)}
```

Comment:
you should name function params with 'on' thats react convention
 onSelect={() => handleSelect(item.id)}
and don't use inline function at all if possible should be, 
onSelect={handleSelect}

so you may need to create your own component to wrap FilterChip if its from library
MyFilterChip({id, onSelect, ...}) = {
const handleSelect = () => onSelect(id)
return <FilterChip .... handleSelect = {handleSelect}>
}